bin/main

```diff
@@ -133,21 +84,21 @@ while want_to_play == 'Y'
 
     flag = 0
     while flag.zero?
-      print "\nPlayer 2 make your move = "
+      print "\n#{p2.name}-#{p2.symbol} make your move = "
       p2_move = gets.chomp
-      if validmove(p2_move.to_i, p1_arr, p2_arr)
+      if Game.validmove(p2_move.to_i, p1.moves, p2.moves)
         flag = 1
       else
         puts "\nThat was an Invalid Move. Input move again\n"
       end
     end
 
-    p2_arr.push(p2_move.to_i)
+    p2.moves.push(p2_move.to_i)
 
-    board(p1_sym, p2_sym, p1_arr, p2_arr)
+    Board.print_board(p1.symbol, p2.symbol, p1.moves, p2.moves)
 
-    if checkwin(p2_move.to_i, p2_arr)
-      puts "\nPlayer2 CONGRATULATIONS!!! you are the WINNER\n"
+    if Game.checkwin(p2_move.to_i, p2.moves)
+      puts "\n#{p2.name} CONGRATULATIONS!!! you are the WINNER\n"
       break
     end
```

Comment:
[Please fix]
- [ ] DRY rule should be respected. you are doing exactly the same thing for player one and player two instead of creating a method and reuse it. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/dry.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)) In order to fix this, you can create a method in your main.rb file that accepts the `player`, `game`, and `board` parameters and move all that logic inside of it.